### PR TITLE
Register luoxh.is-a.dev

### PR DESCRIPTION
**Pull request by Melishy**
Register luoxh.is-a.dev with CNAME record pointing to _github-pages-challenge-Melishy.luoxh.

### CODE REVIEW

File: domains/luoxh.json [hunk 1]
```diff
@@ -0,0 +1,13 @@
+{
+        "owner": {
+           "username": "Melishy",
+           "email": "",
+           "discord": "1053012080812359750",
+           "OWL": "eyJlbmMiOiJBMTI4Q0JDLUhTMjU2IiwiYWxnIjoiUlNBLU9BRVAiLCJraWQiOiJaa1VsRmRqVThiUEstLXVVM2JJR09PVHFYYVFFS1ZINFVXOW53MTR6WTJnIn0.B3hAKGwYwN12MhWyWlMcowtjr_w6FqpinnuTuIXKqUvuTL3ToA17Iy6qIZ6fojNxUsuhtz7fxnE8WkhLsPsVZ08_td5fFMJPThpP4fyqLyFppKjxNnPb4cLSvTec2kFSeyI8FWihG8kfinPeYwps3_rWMUhxQm1lQ-4iBGQ318nthuYs9xVvMsEJH2ftIriSzfA7N4bFJRhJTukB3ssjEH4HEsKDUuuk-TzSJcsSfTLfhWNfqZpIDo-4usv3ZnsWnddXr--nFyUGzRPbuyG_pYb49HJVu2hFGpwFN4AFyPQTX9O70SfRF7r3phm9OFLbvOL5l62Mq-JtBh8hTMKGVw.2jTYnYz8EtdK821s5QzTYQ.BfnuxMM5WPKEQnIYK7-1_p90M8we9BSqSo5hvA7TThM7YIgx-gsfsiOjz--ckrx6H5Ky1vDqXtWhyjDi5wCPti24czng0HpZVZvtHzqMphs.SoyzJiB6WaULLrIMI-Akyg"
+        },
+    
+        "record": {
+            "CNAME": "_github-pages-challenge-melishy.luoxh"
```
Comment (CuteDog5695) [inline domains/luoxh.json:10]: ???
Please read the [docs.](https://www.is-a.dev/docs)

**Review by iostpa — CHANGES_REQUESTED** [review]

**Comment by Melishy** [comment]
I see. Let me fix that

File: domains/luoxh.json (hunk 1 above)
Comment (Melishy) [reply domains/luoxh.json]: Fixed. Please review it.

**Comment by CuteDog5695** [comment]
Can't access the site since it redirects to the is-a.dev domain.
Delete your CNAME file from the websites repo and after this has been merged you can add it back.

**Comment by CuteDog5695** [comment]
Or you can give a preview really

**Comment by Melishy** [comment]
> Can't access the site since it redirects to the is-a.dev domain. Delete your CNAME file from the websites repo and after this has been merged you can add it back.

Uhh, i'm not sure on how to do that. Can you tell on how to delete it?

**Comment by Melishy** [comment]
Oh, i get it!

**Comment by Melishy** [comment]
Removed. Now you can access it.

**Comment by CuteDog5695** [comment]
Yep, I can now access it.
Site seems fine to me so I'll go ahead and merge the PR now.

**Review by iostpa — APPROVED** [review]

**Pull request merged**